src/App.js

```diff
@@ -18,10 +18,10 @@ class App extends Component {
                 "zip": "",
                 "inArea": false,
                 "radius": null,
-                "tuition":[0,70000],
-                "SAT":null,
-                "ACT":null,
-                "ranking": null,
+                "tuition":[0,53000],
+                "SAT":"",
+                "ACT":"",
+                "ranking": "",
```

Comment:
Aren't all three of SAT, ACT, and Ranking numbers?

Also, why not change all these property names to use no quotes, e.g.
```
ACT: "",
SAT: "",
...
```